api/client/samples/similar_regions_Frechet/sr.py

```diff
@@ -0,0 +1,560 @@
+import os, glob
```

Comment:
maybe call this file similar_region.py, so it's more clear that it's main file

Reply:
This IS the main file (the analog of similar_regions.py AND similar_region_state.py put together in the original version). similar_regions_Frechet is fully self-contained.